Packs/DevSecOps/IncidentFields/DevSecOps_After_Commit_ID.json

```
@@ -0,0 +1,30 @@
+{
+    "id": "incident_aftercommitid",
+    "version": -1,
+    "modified": "2020-08-09T11:01:34.448790364+04:00",
+    "name": "After Commit ID",
```

```suggestion
    "name": "Dev Sec Ops After Commit ID",
```
Same in all incident fields/types indicator fields/types